devsetup/scripts/gen-edpm-compute-node.sh

```diff
@@ -199,7 +195,8 @@ chmod +x ${OUTPUT_BASEDIR}/${EDPM_COMPUTE_NAME}-firstboot.sh
 if [ ! -f ${DISK_FILEPATH} ]; then
     if [ ! -f ${CRC_POOL}/${BASE_DISK_FILENAME} ]; then
         pushd ${CRC_POOL}
-        curl -L -k ${CENTOS_9_STREAM_URL} -o ${BASE_DISK_FILENAME}
+        podman run --volume ${CRC_POOL}:/target:rw --rm ${EDPM_IMAGE}
```

Comment:
Podman doesn't have permission to write this file, I need to try this in a real environment to figure it out